src/services/GLSL/vertex_shader_catalog.glsl

```diff
@@ -110,6 +126,11 @@ void main() {
     if (uSizeMajorMapEnabled) {
         vec4 sizeMajor = getValueByIndexFromTexture(uSizeTexture, dataPointIndex);
         float size = sizeMajor.x;
+        if (uIsImagePixelSize) {
+            size = size * uZoomLevel;
```

Comment:
Do we need to apply `getSize` here? Same comment for minor size L157.

Reply:
no, we don't it. I changed the getSize to tuneSymSize that is for making all single-size sources to be consistent with the input value. There is no need for size map sources.

Reply:
@loveluthien When plotting with major size mapping enabled, looks like the size needs to be tuned.
(The below expected sizes are 2px and 1px of y axis.)
![image](https://github.com/user-attachments/assets/e4ccddd9-1afe-4107-8966-635c041b27d8)

Reply:
you are totally right